css/main.css

```diff
@@ -54,7 +54,8 @@ nav .logo {
 nav .user-greet {
   display: none;
```

Comment:
Why do we have `display:none` here?

Reply:
It was already there initially from the initial author for the code base, Should I update or just remove it.